tools/android/packaging/Makefile.in

```diff
@@ -27,7 +27,9 @@ endif
 #this fixes a android ndk fuckup where the paths to
 #prebuilt stuff follow different name schemes for
 #arm and x86
-ifeq ($(findstring x86,$(CPU)),x86)
+ifeq ($(findstring x86_64,$(CPU)),x86_64)
+  ARCH=x86_64
+else ifeq ($(findstring x86,$(CPU)),x86)
```

Comment:
As note: This change is new added.

Found during last review the x86/x86_64 also here checked.